Cython/Coverage.py

```diff
@@ -50,13 +50,22 @@ def _find_dep_file_path(main_file, file_path, relative_path_search=False):
     return canonical_filename(abs_path)
 
 
+class NoCoverConfigurer(CoveragePlugin):
+    exclude_lines = []
+
+    def configure(self, config):
+        type(self).exclude_lines = config.get_option("report:exclude_lines")
```

Comment:
Is this option tested anywhere?

Reply:
Well, yes: It may not be obvious but the plugin does not have any defaults anymore for `report:exclude_lines` so that the fact that the test using `# pragma: no cover` work actually rely on that.

Reply:
Ok, got it, I think. The `FileReporter` now implements `excluded_lines()`, and the `NoCoverConfigurer` picks it up from there. That's not immediately obvious, but yes, it's tested. :)